src/modules/utils/modals/transactionInfo/TransactionInfoCtrl.js

```diff
@@ -18,8 +18,16 @@
                 super($scope);
 
                 this.id = transactionId;
+                this._getData();
+                this.timer = null;
```

Comment:
I would switch those two lines, so that nullifying of the timer was before the method call.